components/service/glean/src/main/java/mozilla/components/service/glean/CommonMetricData.kt

```diff
@@ -51,4 +55,25 @@ interface CommonMetricData {
 
         return true
     }
+
+    /**
+     * Defines the names of the storages the metric defaults to when
+     * "default" is used as the destination storage.
+     * Note that every metric type will need to implement this.
+     */
+    fun defaultStorageDestinations(): List<String>
```

Comment:
nit: Those "getters" can be properties (`val`) too and that's a little bit more idiomatic Kotlin:

```Kotlin
interface CommonMetricData {
    val defaultStorageDestinations: List<String>
}

class SomeMetricData : CommonMetricData {
    override val defaultStorageDestinations: List<String>
        get() = ...
        // or:
        get() {
           ...
       }
    
   // or with a backing field:
   override val defaultStorageDestinations: List<String> = listOf(".....")
}
```

Reply:
Oh, interesting. I had pushed an updated version with this fixed but it's not showing up. I guess I'll have a separate PR for this.